docs/RELEASE_GIT_GRAPH.md

```diff
@@ -0,0 +1,33 @@
+# Expected git graph
+
+![Release git graph](https://ipfs.io/ipfs/bafkreidcmmgmfjoasjimw66mgbl7vnyvjf4ag4y67urj7ejnkkhm7nxx5m)
+
+## Continued branches
+
+- *master* this is development branch
+- *release* this branch list all the release commits
+  It's important that `git show release~$X` shows version releases first.
+
+## Creating this graph:
+
+ 1. Making the RC. From the ref you want to make release do:
+   1. `git checkout -b release-vX.Y.Z` (create the branch this release is gonna be worked on)
```

Comment:
Look at the doc in rendered form.  It isn't indented.  I assume you need to add an extra space.